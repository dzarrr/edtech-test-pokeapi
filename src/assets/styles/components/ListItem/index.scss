.listitem {
  padding: 0.5em;
  transition: background-color 0.2s ease;

  &:hover {
    background-color: #ffc6ae;
    cursor: pointer;
  }
  h4 {
    margin: 0;
  }
  a {
    text-decoration: inherit;
    color: inherit;
  }
}