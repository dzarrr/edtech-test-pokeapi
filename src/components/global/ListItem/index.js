const ListItem = ({ key, children }) => {
  return (
    <div key={key} className='listitem'>
      {children}
    </div>
  )
}

export default ListItem