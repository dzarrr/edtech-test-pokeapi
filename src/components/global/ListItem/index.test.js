import React from 'react'
import { render } from '@testing-library/react'
import ListItem from 'components/global/ListItem'

test('renders element correctly', () => {
  const { asFragment } = render(<ListItem />)

  expect(asFragment()).toMatchSnapshot()
});
