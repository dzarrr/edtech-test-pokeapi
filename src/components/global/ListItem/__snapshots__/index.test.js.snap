// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders element correctly 1`] = `
<DocumentFragment>
  <div
    class="listitem"
  />
</DocumentFragment>
`;
